### ISSUE TITLE
Accordeon 

### ISSUE THREAD

**Issue by StreiffBerenice** [Orange-OpenSource/Orange-Boosted-Angular#60]
@liyokuna - The accordion has some grey lines to separate each accordion. I would be better with black lines (optional)
- The accordion states closed and deployed) should be well vocalised with screen readers (Jaws et NVDA)
<a aria-haspopup="true" aria-expanded="true" href="criteria-ios-conception.html">Critères de conception</a>. We only have the attribute aria-expanded.
See the component accordion : https://deploy-preview-47--ng-boosted.netlify.com/#/documentation/accordion
